DynamicFeatures/app/src/main/res/values-pl/strings.xml

```diff
@@ -18,6 +18,7 @@
 <resources>
     <string name="app_name">Przykład modułów na żądanie</string>
     <string name="force_polish">Wymuś język polski</string>
+    <string name="btn_launch_initial_install">Rozpocznij wstępną instalację</string>
```

Comment:
this means "Start initial installation" but I think you wanted "start initially installed module", so:
"Załaduj wstępnie zainstalowany moduł"

Reply:
Thanks!